scripts/tools/zap/run_zaptool.sh

```diff
@@ -45,9 +45,16 @@ CHIP_ROOT="${SCRIPT_PATH%/scripts/tools/zap/run_zaptool.sh}"
         npm install
     fi
 
+    echo "ARGS: ${ZAP_ARGS[@]}"
+
+    if [[ "${ZAP_ARGS[@]}" == *"all-clusters-app.zap"* ]]; then
```

Comment:
I was unsure if "ZAP ARGS" would contain more than just the zap file name. The script claims to say "only give me the zap file name" but not sure if it would support more.

I will add the slash.

Reply:
`ZAP_ARGS=(-i "$(_get_fullpath "$1")") || ZAP_ARGS=()` so ZAP_ARGS is either the path to the .zap file or nothing, right?